include/SPTDataLoader/module.modulemap

```diff
@@ -0,0 +1,3 @@
+module SPTDataLoader {
+    umbrella "."
```

Comment:
It's a breaking change, but we should probably rename `SPTDataLoaderConvenience.h` to `SPTDataLoader.h` and rename the existing `SPTDataLoader.h` to... something else.